src/compas_rrc/ghpython/components/CompasRrc_MoveToFrame/code.py

```diff
@@ -0,0 +1,50 @@
+"""
+Move robot to the specified frame in cartesian space.
+"""
+from ghpythonlib.component import add_error
+from ghpythonlib.component import add_warning
+from ghpythonlib.componentbase import executingcomponent as component
+
+import compas_rrc as rrc
+from compas_rrc.ghpython import coerce_frame
+
+
+class CompasRrcMoveToFrame(component):
+    def __init__(self):
+        super(CompasRrcMoveToFrame, self).__init__()
+        self.motion_type = rrc.Motion.JOINT
+
+    def RunScript(self, frame, speed, zone, feedback):
+        if not frame:
+            add_warning('Frame is not defined. The component did not run.')
+            return (None)
+
+        if not speed or speed < 0.01:
+            add_warning('Speed must be greated or equals to 0.01 mm/s')
+            return (None)
+
+        if not zone:
+            zone = rrc.Zone.FINE
+
+        frame = coerce_frame(frame)
+        feedback = feedback or rrc.FeedbackLevel.NONE
+        ins = rrc.MoveToFrame(frame, speed, zone, motion_type=self.motion_type, feedback_level=feedback)
+
+        return (ins)
```

Comment:
Are these superlative parentheses or did you mean for this to be a tuple?

Reply:
;) I meant for it to be a tuple because I have the idea that grasshopper expects a tuple of results to assign to the out params, but I will test and remove if that's not the case.